client/src/dataProviders/connectionsDataProviders.js

```diff
@@ -3,18 +3,28 @@ import type {Dispatch} from '../types/reduxTypes'
 import {dispatchReceivedData} from './dataProvidersUtils'
 import {receiveData} from '../actions/sharedActions'
 
-const dispatchEntitiesData = (query: string) => (ref: string, data: any, dispatch: Dispatch) => {
+const dispatchEntitiesData = (query: string) => (
+  ref: string,
+  data: Array<{eid: number}>,
+  dispatch: Dispatch
+) => {
   dispatch(
     receiveData(['connections', 'entities'], {id: query, eids: data.map(({eid}) => eid)}, ref)
   )
 }
 
-const dispatchConnectionData = (eid1: string, eid2: string) => (
+const dispatchConnectionData = (eid1: number | number[], eid2: number | number[]) => (
   ref: string,
-  data: any,
+  data: number[],
   dispatch: Dispatch
 ) => {
-  dispatch(receiveData(['connections', 'detail'], {id: `${eid1}-${eid2}`, ids: data}, ref))
+  dispatch(
+    receiveData(
+      ['connections', 'detail'],
+      {id: `${eid1.toString()}-${eid2.toString()}`, ids: data},
```

Comment:
I think using `toString` is pointless in template literals, isn't it? Also further in this file.

Reply:
Yes, but flow complains when you autocast `number[]` to string :man_shrugging: